### PR TITLE
Dependencies

### PR DESCRIPTION
**Pull request by adeelq**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -8,8 +8,32 @@
     "AFL-3.0"
   ],
   "require": {
-    "php": "~5.5.0|~5.6.0|~7.0.0",
-    "magento/framework": "100.*"
+    "php": "~5.6.0|~7.2.0",
```
Comment (cdiacon) [inline composer.json]: @adeelq   replace with :   "php": "~5.6.0|~7.0.0|~7.1.0",
otherwise will not work with 7.1.* and 7.0.*

**Comment by adeelq** [comment]
@so remove changes from composer..fine to have it in module.xml?

**Comment by cdiacon** [comment]
module.xml is fine, but you can't install the extension anymore using composer as it can't find the dependency defined

**Comment by adeelq** [comment]
cool

**Pull request merged**